src/Serilog/Properties/InternalsVisibleTo.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Runtime.CompilerServices;
 
-#if DOTNET5_4
+#if DOTNET5_4 || DOTNET5_1
```

Comment:
I'm thinking we'll ditch the unsigned version as part of #589 - probably not worth spending time on right now...